videos/WoT-Explainer-Video/WOTMotion.de.vtt

```diff
@@ -0,0 +1,251 @@
+im Bereich der Landwirtschaft.
+
+21
+00:01:19.500 --> 00:01:23.000
+Die Entwickler sind glücklich 
+viele Informationen zu finden
+
+22
+00:01:23.500 --> 00:01:27.000
+von WoT-Tutorials wie auf thingweb.io, 
+YouTube, und GitHub.
+
+23
+00:01:27.500 --> 00:01:29.000
+Mit node-wot können verschiedene 
+Protokollanbindungen
+
+24
+00:01:29.500 --> 00:01:34.000
+wie HTTP, MQTT, CoAP 
+und Modbus,
+
+25
+00:01:34.500 --> 00:01:39.000
+sowie einem Browser-Bundle, können die Entwickler leicht eine 
+eine Full-Stack-IoT-Lösung entwickeln.
+
+26
+00:01:39.500 --> 00:01:42.000
+Ihre Lösung verbindet 
+alle Geräte
+
+27
+00:01:42.500 --> 00:01:45.000
+und beinhaltet eine großartige Benutzeroberfläche 
+die mit Web-Technologie aufgebaut ist.
+
+28
+00:01:45.500 --> 00:01:49.000
+Anstatt eine einmalige 
+proprietäre Lösung zu schaffen,
+
+29
+00:01:49.500 --> 00:01:54.000
+verwenden die Entwickler die WoT-Methodik 
+um IoT-Builiding Blocks zu erstellen.
+
+30
+00:01:54.500 --> 00:01:59.000
+Diese Building Blocks erweitern 
+ihr bestehendes Portfolio an Web-Technologien.
+
+31
+00:01:59.500 --> 00:02:03.000
+Thing Description, 
+WoT-Anwendungsvorlagen,
+
+32
+00:02:03.500 --> 00:02:08.000
+und IoT-Semantik sind jetzt im gleichen Portfolio 
+von wiederverwendbaren Modulen
+
+33
+00:02:08.500 --> 00:02:13.000
+wie HTML-Seiten, JavaScript-Pakete 
+oder CSS aus früheren Web-Projekten.
+
+34
+00:02:13.500 --> 00:02:18.000
+Das Hauptelement 
+der WoT-Methodik
+
+35
+00:02:18.500 --> 00:02:21.000
+ist die Thing Description, 
+auch bekannt als TD.
+
+36
+00:02:21.500 --> 00:02:26.000
+Die TD ist vergleichbar 
+mit einer index.html für Things. 
+
+37
+00:02:21.500 --> 00:02:30.000
+Verschiedene Fragen können beantwortet werden
+ durch die Informationen in der TD,
+
+38
+00:02:30.500 --> 00:02:40.000
+wie Identifikation, Datenorganisation, Zugriff, Einheiten,
+Typen, Interaktionen, Sicherheit oder Beziehungen.
+
+39
+00:02:40.500 --> 00:02:44.000
+Klassische IoT-Projekte 
+verursachen in der Regel hohe Kosten,
+
+40
+00:02:44.500 --> 00:02:47.000
+insbesondere aufgrund 
+durch den hohen Integrationsaufwand
+
+41
+00:02:47.500 --> 00:02:51.000
+der unterschiedlichen Datenmodelle 
+und Protokollen der Geräte bzw. Dienste.
+
+42
+00:02:51.500 --> 00:02:57.000
+Im Gegensatz dazu erstellen die Entwickler mit WoT 
+die IoT-Lösung in nur wenigen Sprints,
+
+43
+00:02:57.500 --> 00:03:03.000
+indem sie WoT TDs, Protokollbindungen 
+und Scripting mit ihrer Web-Expertise kombinieren. 
+
+44
+00:03:03.500 --> 00:03:07.000
+Dies führt zu einer Reduzierung der Entwicklungs 
+und Wartungskosten,
+
+45
+00:03:07.500 --> 00:03:09.000
+spart Zeit 
+und Geld.
+
+46
+00:03:09.500 --> 00:03:11.000
+Die Entwickler gewinnen Vertrauen 
+in IoT
+
+47
+00:03:11.500 --> 00:03:17.000
+und betrachten sich von nun an als Experten 
+sowohl in Web- als auch in IoT-Technologien.
+
+48
+00:03:17.500 --> 00:03:22.000
+Auf Basis von WoT konnte das IoT-Projekt des Landwirts 
```

Comment:
I think "Landwirts" is fine.. but we used "Landwirtes" before and so i think we should use it here also.